### ISSUE TITLE
IDEA: Implementing pronote server back-end in python

### ISSUE THREAD

**Issue by nouni19** [bain3/pronotepy#122]
I was thinking, what if pronotepy included a back-end for pronote too? I could help with all of that if there's a reliable source for the protocol

**Comment by bain3** [comment]
This is a big project. I don't think it's a great idea to put it in pronotepy directly though. Most people won't ever use it and it would be a big burden when maintaining. 
Also I'm not sure how to get the precise inner-workings of the PRONOTE server to replicate it. Currently all of the protocol was reverse engineered and from what we've seen index-education aren't really happy libraries exist, so they won't help. I'm honestly surprised they didn't send me and a few other people an email too when Litervan's pronote-api was taken down.

tl;dr not really feasible, technically complex
